src/client/components/DeliveryInfo/DeliveryInfo.component.js

```diff
@@ -31,7 +31,7 @@ export default function DeliveryInfo({ editMode, vertDisplay, user }) {
   return (
```

Comment:
Jai, above in line 6 , zip should be zipcode to get value, as you get user as props.

Reply:
Done